scripts/file_resume.js

```diff
@@ -55,10 +96,13 @@ window.addEventListener("DOMContentLoaded", function() {
             return false;
         }
         if(validate(file.name)) {
-            // won't work if we disable browse button so hide it
+            // won't work if we disable the buttons so hide them
             document.getElementById("old_browse").style.display = "none";
-            document.getElementById("old_browse").disabled = true;
+            document.getElementById("old_submit").style.display = "none";
             showProgress(uploadMessages.working);
+            ev.preventDefault();
+            fileHash(file)
```

Comment:
do we want a .catch?

Reply:
Good point. We should. Perhaps just upload an empty string in this case and let the server say there was no checksum provided.